ha/core/event_manager/event_manager.py

```diff
@@ -135,8 +135,8 @@ def _store_component_key(self, key: str, resource_type: str, val: list) -> None:
         '''
            Perform actual confstore store operation for component keys
            Ex:
-           key: cortx>ha>v1>events>subscribe>sspl
-           confstore value: ['enclosure:sensor:voltage>failed', ...]
+           key: cortx>ha>v1>events>subscribe>hare
+           confstore value: ['node>server>failed', 'disk>online' ...]
```

Comment:
Keep it uniform for all resource types. For disk it can be 'disk>ALL>online' if we decide on using ALL for all functional types for given resource type (disk can have data and metadata as two functional types).

Reply:
ALL processing can be addressed in separate ticket for all resource_types. Do not need to do it in this ticket.